evaluation/config-demo.jsonc

```
@@ -4,7 +4,7 @@
     "host": "0.0.0.0",
     "port": 8000
   },
-  /* AMQP endpoint */
+  /* Celery broker endpoint */
   "amqp": {
```

Could we do
```suggestion
  "celery": {
```
(the same in evaluation/config-isolated-services.jsonc, ts/backend/config/config.jsonc and ts/backend/src/app/features/config/defaults.mts)

Or would this be too impactful? (E.g. helm deployment)